app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue

```diff
@@ -82,8 +96,10 @@ export default {
   data() {
     return {
       selectedAgents: [],
+      autoAssignment: false,
       isUpdating: false,
       isAgentListUpdating: false,
+      isAutoAssignmentUpdating: false,
```

Comment:
I would recommend to move this out of the the component, please see the uiFlags usage in the inboxes module